src/main.h

```diff
@@ -136,7 +136,7 @@ extern bool fPruneMode;
 /** Number of MiB of block files that we're trying to stay below. */
 extern uint64_t nPruneTarget;
 /** Block files containing a block-height within MIN_BLOCKS_TO_KEEP of chainActive.Tip() will not be pruned. */
-static const signed int MIN_BLOCKS_TO_KEEP = 288;
+static const unsigned MIN_BLOCKS_TO_KEEP = 288;
```

Comment:
We never use `unsigned` without type in the code anywhere else, let's just make this `unsigned int`.